### ISSUE TITLE
Present user indication to where the coordinates are.

### ISSUE THREAD

**Issue by solonovamax** [AMereBagatelle/MCG#19]
Presnt the user with an indicator to help them locate the coordinates without relying on f3, or mods like minihud.

For example, JustMap's waypoint renderer:
https://github.com/Bulldog83/JustMap/blob/1.16.2-ulimited/src/main/java/ru/bulldog/justmap/client/render/WaypointRenderer.java#L85-L220

This would be very helpful for helping users return to previously marked coordinates.

**Comment by AMereBagatelle** [comment]
This has been in the back of my mind to do, however I haven't figured out the best way to do it.  I really wanted to avoid the idea of waypoints per se, as they create a lot of clutter onscreen, but there definitely needs to be an indicator of at least what direction to go.
Possible solution would be to just render a compass above the HUD for the overlay text, or utilize the compass item in some way.

**Comment by solonovamax** [comment]
A compass works, but how about a beacon beam at the desired location?

It'd be visually in the style of minecraft, and would only apply to the *selected* waypoint

**Comment by SimplyTolex** [comment]
> and would only apply to the _selected_ waypoint

yes, it definitely should only apply to a selected since otherwise it's just an minimap without a minimap and at that point why even bother
